public/content/wrapped-ether/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: What is Wrapped Ether (wETH)
+description: An introduction to Wrapped Ether (wETH)—an ERC20-compatible version of Ether (ETH). 
+lang: en
+---
+
+# Wrapped Ether (wETH)
+
+[Ether](/eth) (ETH) is the native token on Ethereum, and is used for staking, as a currency and paying for gas fees for computation. However, ETH has limitations in how it can interact with assets and applications built on top of Ethereum, like ERC-20 tokens. In order for ETH to be able to interact with ERC-20 tokens, it must conform with the ERC-20 standard.
+
+To bridge this gap, wrapped ETH (wETH) was created. Wrapped ETH is a smart contract that lets you deposit 1 ETH into the contract and receive 1 minted wETH that conforms to the ERC-20 token standard. wETH is a representation of ETH that allows you to interact with it as an ERC-20 token, not as the native asset ETH. You will still need native ETH to pay for gas fees, so make sure you save some when depositing. 
+
+You are able to unwrap wETH for ETH by using the wETH smart contract. You can deposit 1 wETH back into the wETH smart contract, and you will receive 1 ETH. The wETH deposited is then burned and taken out of the circulating supply of wETH.
+
+Roughly 3% of the circulating ETH supply is locked in the wETH token contract making it one of the most used smart contracts. wETH is especially important with users interacting with applications in decentralized finance (DeFi).
+
+## Why do we need to wrap ETH as an ERC-20? {#why-do-we-need-to-wrap-eth} 
+
+[ERC-20](/developers/docs/standards/tokens/erc-20/) defines a standard interface for fungible tokens, so anyone can create tokens that interact seamlessly with applications and tokens that use this standard in Ethereum's ecosystem. Since ETH predates the ERC-20 standard, ETH doesn't conform to this specification. This means you can't easily exchange ETH for other ERC-20 tokens or use ETH in apps using the ERC-20 standard. Wrapping ETH gives you the opportunity to do the following:
```

Comment:
```suggestion
[ERC-20](/developers/docs/standards/tokens/erc-20/) defines a standard interface for fungible tokens, so anyone can create tokens that interact seamlessly with applications and tokens that use this standard in Ethereum's ecosystem. Since **ETH predates the ERC-20 standard**, ETH doesn't conform to this specification. This means **you can't easily** exchange ETH for other ERC-20 tokens or **use ETH in apps using the ERC-20 standard**. Wrapping ETH gives you the opportunity to do the following:
```